src/scss/styles-1-large-main.scss

```diff
@@ -671,3 +671,9 @@ footer p {
    color: black;
 }
 
+.css-1y7sxf6, [data-css-1y7sxf6] {
```

Comment:
can we define a better name for the classes? Or is this any name convention I do not know about?

Reply:
fixed